standards/rmrk2.0.0/entities/base.md

```diff
@@ -0,0 +1,200 @@
+# Base
+
+A base is a meta-entity, stored on-chain or off chain. It is not an NFT, but can be thought of as
+the interface or abstract class for an NFT's render.
+
+A base is a JSON object with three properties: `id`, `type`, and `parts`. There is an implied
+`owner` field, matching the address that created the base.
+
+Example:
+
+```json=
+{
+    "id": "kanaria_superbird",
+    "type": "svg",
+    "parts": [
+        {
+            "bg": {
+                "type": "fixed",
+                "z": 0,
+                "src": "ipfs://ipfs/hash"
+            },
+            "gem_1": {
+                "type": "slot",
+                "collections": ["id-of-genesis-trait-crystals-LEGENDARY"],
+                "unequip": "unequip",
+                "z": 1
+            },
+            "gem_2": {
+                // ...
+            },
+            "gem_3": {
+                // ...
+            },
+            "gem_4": {
+                // ...
+            },
+            "gem_5": {
+                // ...
+            },
+            "wing_1_back": {
+                "type": "fixed",
+                "z": 1,
+                "src": "ipfs://ipfs/hash"
+            },
+            "wing_1_front": {
+                "type": "fixed",
+                "z": 3,
+                "src": "ipfs://ipfs/hash2"
+            },
+            "wing_1_slot": {
+                "type": "slot",
+                "equippable": ["id-of-genesis-legendaries", "id-of-genesis-rares", "id-of-genesis-epics", ...],

```

Comment:
In this example you use both `equippable` and `collections` as array of allowed collections?